docpages/advanced_reference/automating-with-jenkins.md

```diff
@@ -34,30 +33,30 @@ cmake --build build/
 
 \note This script does not make use of multiple threads. If you know how to do this and you would like to use threads, then feel free to. However, I would be careful not to use all your threads as Jenkins may dislike this.
 
-This script will build your project for you and also setup CMake if you've deleted the build directory, allowing you to easily refresh CMake. You can change this to a build parameter if you want, meaning you can hit `Build with Parameters` and state what you'd like to do (This would require the script to be changed so only do that if you know what you're doing)!
+This script will build your project for you and also set up CMake if you've deleted the build directory, allowing you to easily refresh CMake. You can change this to a build parameter if you want, meaning you can hit `Build with Parameters` and state what you'd like to do. (This would require the script to be changed so only do that if you know what you're doing!)
```

Comment:
Yeah, again, brackets can be removed. Silly thing from me haha.

Reply:
Sure.